pkg/debug/checkups/appindicator.go

```diff
@@ -0,0 +1,50 @@
+package checkups
```

Comment:
This duplicates https://github.com/kolide/launcher/blob/main/pkg/debug/checkups/gnome-extensions.go I don't think we need both?

Reply:
I was just leaving it there for a bit to compare, it's removed now.